lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb

```diff
@@ -168,12 +168,25 @@
     "    gamma_options['interp_fraction'] * gamma_options['max_gamma'])\n",
```

Comment:
<p>Can we remove this "flag probability density" instead just pick one or the other of the options and run with that?</p><p> </p><p>You can pick which one to use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Good idea. Will remove it now.

Reply:
<p>"flag probability density" has been removed. You can check :-)</p>